audio/openal-soft/Portfile

```diff
@@ -93,3 +96,15 @@ variant tests description {Install test programs} {
 
 livecheck.type          regex
 livecheck.regex         /${name}-(\[0-9.\]+)${extract.suffix}
+}
+
+subport ${name}-native-tools {
+    cmake.source_dir ${worksrcpath}/native-tools
+    destroot {
+        xinstall -d -m 0755 ${destroot}${prefix}/libexec/openal-soft-native-tools/bin
+        foreach bin {bin2h bsincgen} {
+            xinstall -m 0755 ${cmake.build_dir}/${bin} ${destroot}${prefix}/libexec/openal-soft-native-tools/bin
+        }
```

Comment:
There's no reason to use a foreach loop, is there? Can't it just be:
```
xinstall -W ${cmake.build_dir} bin2h bsincgen ${destroot}${prefix}/libexec/openal-soft-native-tools/bin
```

Reply:
The `foreach` has now been removed.